scripts/test.js

```diff
@@ -30,40 +31,15 @@ function areComponentsPackaged() {
     return false;
 }
 
-const FORM_CONTROL_TEST_DEPENDENCIES = ['checkbox', 'dropdown', 'radio-button', 'rangeslider', 'switch'];
-
-function copyTestDependencies(component, dependencies) {
-    for (let dependency of dependencies) {
-        const componentPackageName = `${dependency}.development.js`;
-        fs.copyFileSync(
-            path.join(COMPONENTS_FOLDER, dependency, 'umd', componentPackageName),
-            path.join(TESTS_FOLDER, component, componentPackageName));
-    }
+function linkDependencies() {
+    execSync('npm run link', {cwd: ROOT_FOLDER, stdio: 'inherit'});
+    linkSingleComponent('', TESTS_FOLDER);
 }
 
-function test(rebuild, browsersArg) {
-    if (rebuild) execSync('npm run rebuild', { cwd: path.join(__dirname, '../'), stdio: 'inherit' });
+function test(rebuild, browsersArg, link) {

```

Comment:
Should `link` argument need to has default value?

Reply:
Not mandatory because we call test from main where we define link as true, but definitely good to have default value, in case someone else decides to use the test function. Added.